mitosheet/mitosheet/enterprise/mito_log_uploader.py

```diff
@@ -0,0 +1,114 @@
+import json
+import pprint
+import time
+from typing import Any, Optional, List, Dict
+import requests
+
+def preprocess_log_for_upload(log_event: str, log_params: Dict[str, Any]) -> Optional[Dict[str, Any]]:
```

Comment:
This approach is currently:
1. Take our current logs that we generate
2. Filter them to make them into the format that we want

This means that our enterprise logging approach is like sorta dependent on our current logging approach (which we have iterated in the past). 

Another way to do this would be to go farther up-the-chain to the parameters to the `log` call.

No need to change anything, just calling this out for info/thought sharing reasons.